modules/olmv1-installing-an-operator.adoc

```diff
@@ -170,7 +174,7 @@ items:
     uid: c54237be-cde4-46d4-9b31-d0ec6acc19bf
   spec:
     channel: latest
-    installNamespace: openshift-operators
+    installNamespace: pipelines
```

Comment:
@michaelryanpeter the output example miss serviceAccount
and the status condition outout miss the Unpacked.

so, I suggest to execute it to get latest output for `oc get clusterextension pipelines-operator -o yaml`

here is the example I did. FYI
```console
[root@preserve-olm-env2 openshift-tests-private]# oc apply -f /tmp/-2pbellh8config.json.stdout
clustercatalog.catalogd.operatorframework.io/clustercatalog-75493 created
[root@preserve-olm-env2 openshift-tests-private]# oc get clustercatalog.catalogd.operatorframework.io/clustercatalog-75493 -o yaml
apiVersion: catalogd.operatorframework.io/v1alpha1
kind: ClusterCatalog
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"catalogd.operatorframework.io/v1alpha1","kind":"ClusterCatalog","metadata":{"annotations":{},"name":"clustercatalog-75493"},"spec":{"source":{"image":{"ref":"quay.io/olmqe/nginx-ok-index:vokv3283"},"type":"image"}}}
  creationTimestamp: "2024-09-25T03:06:50Z"
  finalizers:
  - catalogd.operatorframework.io/delete-server-cache
  generation: 1
  name: clustercatalog-75493
  resourceVersion: "83595"
  uid: ecfb980c-9e4c-44c7-8c0f-2a9eb654ea3d
spec:
  source:
    image:
      ref: quay.io/olmqe/nginx-ok-index:vokv3283
    type: image
status:
  conditions:
  - lastTransitionTime: "2024-09-25T03:06:57Z"
    message: ""
    reason: UnpackSuccessful
    status: "True"
    type: Unpacked
  contentURL: https://catalogd-catalogserver.openshift-catalogd.svc/catalogs/clustercatalog-75493/all.json
  observedGeneration: 1
  phase: Unpacked
  resolvedSource:
    image:
      lastPollAttempt: "2024-09-25T03:06:57Z"
      ref: quay.io/olmqe/nginx-ok-index:vokv3283
      resolvedRef: quay.io/olmqe/nginx-ok-index@sha256:a3de4aa1f9542b936ac0a734ef1822336cf465e2d75c85d70dbebd6464048800
    type: image
[root@preserve-olm-env2 openshift-tests-private]# oc apply -f /tmp/-jeevovmjconfig.json.stdout
clusterextension.olm.operatorframework.io/enough-75493 created
[root@preserve-olm-env2 openshift-tests-private]# oc get clusterextension.olm.operatorframework.io/enough-75493 -o yaml
apiVersion: olm.operatorframework.io/v1alpha1
kind: ClusterExtension
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"enough-75493"},"spec":{"channel":"alpha","installNamespace":"ns-75493","packageName":"nginx-ok-v3277775493","serviceAccount":{"name":"sa75493"},"upgradeConstraintPolicy":"Enforce","version":"\u003e=0.0.1"}}
  creationTimestamp: "2024-09-25T03:07:42Z"
  finalizers:
  - olm.operatorframework.io/cleanup-unpack-cache
  generation: 1
  name: enough-75493
  resourceVersion: "84009"
  uid: 411f44a8-ee0d-4b3c-a39b-09820263dabc
spec:
  channel: alpha
  installNamespace: ns-75493
  packageName: nginx-ok-v3277775493
  serviceAccount:
    name: sa75493
  upgradeConstraintPolicy: Enforce
  version: '>=0.0.1'
status:
  conditions:
  - lastTransitionTime: "2024-09-25T03:07:42Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: Deprecated
  - lastTransitionTime: "2024-09-25T03:07:42Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: PackageDeprecated
  - lastTransitionTime: "2024-09-25T03:07:42Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: ChannelDeprecated
  - lastTransitionTime: "2024-09-25T03:07:42Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: BundleDeprecated
  - lastTransitionTime: "2024-09-25T03:07:42Z"
    message: resolved to "quay.io/olmqe/nginx-ok-bundle:v3277775493"
    observedGeneration: 1
    reason: Success
    status: "True"
    type: Resolved
  - lastTransitionTime: "2024-09-25T03:07:42Z"
    message: 'unpack successful: '
    observedGeneration: 1
    reason: UnpackSuccess
    status: "True"
    type: Unpacked
  - lastTransitionTime: "2024-09-25T03:07:43Z"
    message: Installed bundle quay.io/olmqe/nginx-ok-bundle:v3277775493 successfully
    observedGeneration: 1
    reason: Success
    status: "True"
    type: Installed
  installedBundle:
    name: nginx-ok-v3283-75493.v0.0.1
    version: 0.0.1
  resolvedBundle:
    name: nginx-ok-v3283-75493.v0.0.1
    version: 0.0.1

```

Reply:
I haven't been able to get any of the Red Hat catalogs to work, so I have not been able to test these procedures with the Pipelines Operator. I tried to update the output in the doc to match your example.

https://github.com/openshift/openshift-docs/pull/82245/commits/dcc24db551944706efa989b0b5fd9d37ebce362f#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R175-L232

Once we can use private registries in v1, I will redo all of the `.Example outputs` to match the new statuses.

Is this good enough for now?

Reply:
ok. you could update it late.